.drone/README.md

```diff
@@ -18,3 +22,21 @@ Alternatively, run the following `make` command from the project root directory:
 ```
 make drone
 ```
+
+## Drone CI environment
+
+The test coverage step requires the `jq` tool to run. The `jq` tool is provided by the `Dockerfile` in this directory. If you need to update the image, make your changes to the `Dockerfile` and run the following to push the new image to GCR:
```

Comment:
I'm a little confused, is jq a thing that needs to be installed?  does `make drone-env` install it?  I'm unsure whether these steps will install jq, or if jq is needed for these steps.

Reply:
Oh yea, I can see where the confusion comes from--I'll try to clear it up